rust/src/transformations/make_stable_expr/expr_discrete_quantile_score/mod.rs

```diff
@@ -56,9 +56,21 @@ where
     if active_series.nullable {
         return fallible!(
             MakeTransformation,
-            "Quantile estimation requires non-null inputs"
+            "Quantile estimation requires non-null inputs. Try using `.fill_null(x)` or `.drop_null()` first."
         );
     }
+    let nan = match active_series.dtype() {
```

Comment:
This is where the "not necessary for privacy, but good for utility" check is added.